feed/views.py

```diff
@@ -498,12 +498,12 @@ def list(self, request):
         if request.user.is_superuser:
             queryset = StrategicObjective.objects.all()
         else:
-            user_org = TolaUser.objects.get(user=request.user).organization
-            queryset = StrategicObjective.objects.all().filter(country__organization=user_org)
+            user = TolaUser.objects.get(user=request.user)
+            queryset = StrategicObjective.objects.all().filter(country=user.country, organization=user.organization)
```

Comment:
I think that you don't need the all() if you use filter()

Reply:
You're right. We don't need to use the all() but we're using in many other places. Maybe, we should change them in the future as well.